parsl/executors/high_throughput/executor.py

```diff
@@ -545,10 +567,6 @@ def submit(self, func, resource_specification, *args, **kwargs):
         Returns:
               Future
         """
-        if resource_specification:
```

Comment:
might be a better user experience to check for unknown keywords in the resource dictionary, rather than ignore this check completely.

Reply:
This check was placed since HTEX did not support `resource_specification`. We are not doing any validation of  the resource_specification now, but I'm hoping to do that in a later round of work.

Reply:
From discusssion with @benclifford, we want to do a basic check to confirm that the user supplied spec only contains keys that are supported.